testgres/node.py

```diff
@@ -856,7 +863,24 @@ def backup(self, **kwargs):
 
         return NodeBackup(node=self, **kwargs)
 
-    def replicate(self, name=None, **kwargs):
+    def create_replication_slot(self, slot_name, dbname=None, username=None):
```

Comment:
Should we allow user to choose his own `slot_name`? If not, it's better to inline this method.

Reply:
If we don't allow this, then it could contradict with logical replication interface (#42) which allows user to choose the subscriber name (which implicitly creates logical replication slot) voluntarily.

Speaking of inlining, I can call `create_replication_slot()` from `replicate()` if `slot_name != None` so that user won't need to call it explicitly. It would be more convenient for user.

Reply:
@zilder You're right, of course. Maybe we'll add a default name generator later.

> Speaking of inlining, I can call create_replication_slot() from replicate() if slot_name != None

Sure, that would be nice.

Reply:
> It's better to check if slot exists, though.

True